src/app/plugins/register.plugin.ts

```diff
@@ -16,16 +16,35 @@ export class RegisterPlugin extends Plugin {
   }
 
   public async execute(message: IMessage, args?: string[]) {
```

Comment:
let's re-work this to use some regex to tell between an argument that is a classchannel and an arg that is a category

Reply:
I don't have much experience with regex. If you have a specific regex that applies I can implement that.